projects/commudle-admin/src/app/feature-modules/labs/components/labs/labs.component.scss

```diff
@@ -3,6 +3,45 @@
 .labs-home {
   background-color: $white;
   min-height: 100vh;
+
+  .page-container {
+    .body {
+      .popular-tags {
+        display: flex;
+        gap: 32px;
+        padding-left: 156px;
+        font-family: $primary-font-family;
+        div {
+          font-size: 18px;
+          p {
+            cursor: pointer;
```

Comment:
Just a suggestion, instead of giving cursor pointer in the CSS, you can give the class 'clickable' in the corresponding HTML tag